internal/venice-client-common/src/main/java/com/linkedin/venice/httpclient5/HttpClient5Utils.java

```diff
@@ -127,6 +129,7 @@ public CloseableHttpAsyncClient build() {
                     .setResponseTimeout(Timeout.ofMilliseconds(requestTimeOutInMilliseconds))
                     .setConnectionRequestTimeout(CONNECT_TIMEOUT_IN_MILLISECONDS)
                     .build())
+            .setUserTokenHandler((route, context) -> null)
```

Comment:
This is a very interesting finding... so does it mean that without adding this line, the TLS handshake would be redone on every request?!

Reply:
Right, thats what the experiments show.

Reply:
On every request???
This shouldn't be possible, otherwise FC latency will be very bad, and it can't achieve 2ms at p99 for single key lookup.

Reply:
Add this is for HTTP/1.1, not HTTP/2, right?
I was overly- aranoid..

Reply:
Right its only for HTTP/1, not 2.  I verified for H2 we do only 1 handshake.

Reply:
And is this line useful at all?
Maybe the `connectionReuseStrategy` is good enough.

Reply:
hmm.. I could try that, but doubt thats the case.